src/Auxiliary/Aux_TakeNote.cpp

```diff
@@ -1476,9 +1488,12 @@ void Aux_TakeNote()
 #     ifdef SUPPORT_LIBYT
       fprintf( Note, "Parameters of YT Inline Analysis\n" );
       fprintf( Note, "***********************************************************************************\n" );
-      fprintf( Note, "YT_SCRIPT                       %s\n",      YT_SCRIPT  );
-      fprintf( Note, "YT_VERBOSE                      %d\n",      YT_VERBOSE );
-      fprintf( Note, "YT_FIG_BASENAME                 %s\n",      YT_FIG_BASENAME );
+      fprintf( Note, "YT_SCRIPT                           %s\n",      YT_SCRIPT  );
+      fprintf( Note, "YT_VERBOSE                          %d\n",      YT_VERBOSE );
+      fprintf( Note, "YT_FIG_BASENAME                     %s\n",      YT_FIG_BASENAME );
+#     ifdef LIBYT_JUPYTER
+      fprintf( Note, "LIBYT_JUPYTER_USE_CONNECTION_FILE   %s\n",      LIBYT_JUPYTER_USE_CONNECTION_FILE ? "ON" : "OFF" );
```

Comment:
Replace it with `fprintf( Note, "LIBYT_JUPYTER_USE_CONNECTION_FILE   %d\n",      LIBYT_JUPYTER_USE_CONNECTION_FILE );` to match other boolean variables like `YT_VERBOSE`.

Reply:
Done.